### ISSUE TITLE
Add Notes box to the Create journal suspense screen

### ISSUE THREAD

**Issue by zoecrosby** [LocalGovIMS/localgov-ims#231]
Users have asked for the ability to be able to add a note at the same time as re-allocating the suspense item.

In suspense could we have the notes box adding to when we journal a transaction to save having to go in once to put the note on and then again to journal it. It takes quite a while to load and save the note bit so thought could save a bit of time in the long run?

**Comment by bmbc-dev-detec** [comment]
Hi @zoecrosby - the obvious issue here is that space on the screen is very limited, which makes adding any new fields to this table style layout somewhat tricky (if I've correctly understood what you're suggesting).

However, I've been having a look into the performance issue which you've noted. 
The loading of the notes pages seems pretty quick, but loading the suspense list is slow even with just a few records in the database. I've investigated and it's caused by the nesting of 2 actions within the list view which render two partial views used as modals for journals and credit notes.
Both of these views require quite a few DB lookups.

We have accessibility issues related to the modals when the user zooms in, and I think we're only going to resolve those by replacing the modals with full views. If we do that, the need to load these partials views disappears and the performance should increase. (And I'm not suggesting replacing the modals is a simple job).

**Comment by zoecrosby** [comment]
@bmbc-dev-detec agreed the modal screen is busy, so happy to add this to the back log to be done as we change the modal screens to proper pages
